modules/distribution/resources/operation_policies/specifications/opaPolicy.json

```diff
@@ -0,0 +1,94 @@
+{
+  "category": "Mediation",
+  "name": "opaPolicy",
+  "displayName": "Validate Request With OPA Policy",
+  "description": "With this policy, user can validate requests based on the OPA policy engine",
+  "multipleAllowed": true,
+  "policyAttributes": [
+    {
+      "name": "serverUrl",
+      "displayName": "OPA server URL",
+      "description": "OPA server's URL",
+      "validationRegex": "([\\w+]+\\:\/\/)?([\\w\\d-]+\\.)*[\\w-]+[\\.\\:]\\w+([\/\\?\\=\\&\\#\\.]?[\\w-]+)*\/?",
+      "type": "String",
+      "required": true
+    },
+    {
+      "name": "accessKey",
+      "displayName": "Access Key",
+      "description": "Access key for validation request",
+      "validationRegex": "([a-zA-Z\\d_][a-zA-Z\\d_\\-\\ ]*)$",
```

Comment:
Lets allow any value for access key, feels it is safe and I think we do not want to validate this value

Reply:
Fixed